apps/web/src/services/asset-inventory/collector/create-collector/modules/AttachedServiceAccountForm.vue

```diff
@@ -0,0 +1,181 @@
+                       :invalid="invalidState.selectedAttachedServiceAccount"
+                       :invalid-text="invalidTexts.selectedAttachedServiceAccount"
+                       :valid="state.isAttachedServiceAccountValid"
+                       :valid-text="$t('Good')"
+                       :required="true"
+        >
+            <p-radio-group class="attached-service-account-radio-group">
+                <p-radio v-for="(item) in attachedServiceAccountList"
+                         :key="`${item.name}`"
+                         :value="item.name"
+                         :selected="state.selectedAttachedServiceAccountType"
+                         @change="handleChangeAttachedServiceAccountType"
+                >
+                    {{ item.label }}
+                </p-radio>
+            </p-radio-group>
+            <p-select-dropdown class="attached-service-account-dropdown"
+                               :selected="state.selectedAttachedServiceAccountType"
+                               :items="attachedServiceAccountList"
+                               @update:selected="handleChangeAttachedServiceAccountType"
+            />
+            <div v-if="state.selectedAttachedServiceAccountType !== 'all'">
+                <p-field-title class="specific-service-account-dropdown-label"
+                               :label="$t('Specific Service Account')"
+                />
+                <p-filterable-dropdown class="specific-service-account-dropdown"
+                                       :selected="selectedAttachedServiceAccount"
+                                       multi-selectable
+                                       :menu="state.serviceAccountMenu"
+                                       appearance-type="badge"
+                                       @update:selected="setForm('selectedAttachedServiceAccount', $event)"
+                />
+            </div>
+        </p-field-group>
+    </div>
+</template>
+
+<script lang="ts" setup>
+import { computed, reactive, watch } from 'vue';
+import type { TranslateResult } from 'vue-i18n';
+
+import {
+    PFieldGroup, PRadioGroup, PRadio, PFilterableDropdown, PSelectDropdown, PFieldTitle,
+} from '@spaceone/design-system';
+
+import { i18n } from '@/translations';
+
+import { useFormValidator } from '@/common/composables/form-validator';
+
+import { useCollectorFormStore } from '@/services/asset-inventory/store/collector-form-store';
+
+interface Props {
+    title?: TranslateResult;
+}
+
+const emit = defineEmits([
+    'update:isAttachedServiceAccountValid',
+]);
+
+const props = defineProps<Props>();
+const collectorFormStore = useCollectorFormStore();
+
+const attachedServiceAccountList = [
+    {
+        label: 'All', // TODO: translation
+        name: 'all',
+    },
+    {
+        label: 'Specific Service Account', // TODO: translation
+        name: 'specific',
+    },
+];
+
+const state = reactive({
+    selectedAttachedServiceAccountType: 'all',
```

Comment:
suggestion: Please add types too!

```suggestion
    selectedAttachedServiceAccountType: 'all' as 'all'|'specific',
```